From bda26f88e19f8319d7b19672bf41c12e13e4cbe4 Mon Sep 17 00:00:00 2001
From: Sergio Schvezov <sergio.schvezov@canonical.com>
Date: Wed, 7 Sep 2016 12:58:47 +0300
Subject: [PATCH 01/10] adb: remove selinux extensions

* drop useless includes of Android SELINUX extensions
* avoids having to clone another module
* this should be sent upstream

Upstream-Status: Inappropriate

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 adb/file_sync_service.cpp | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/adb/file_sync_service.cpp b/adb/file_sync_service.cpp
index 926dbcfa9b..317e2a346d 100644
--- a/adb/file_sync_service.cpp
+++ b/adb/file_sync_service.cpp
@@ -22,7 +22,6 @@
 #include <dirent.h>
 #include <errno.h>
 #include <log/log.h>
-#include <selinux/android.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
@@ -72,8 +71,6 @@ static bool secure_mkdirs(const std::string& path) {
             if (chown(partial_path.c_str(), uid, gid) == -1) {
                 return false;
             }
-            // Not all filesystems support setting SELinux labels. http://b/23530370.
-            selinux_android_restorecon(partial_path.c_str(), 0);
         }
     }
     return true;
@@ -173,9 +170,6 @@ static bool handle_send_file(int s, const char* path, uid_t uid,
             goto fail;
         }
 
-        // Not all filesystems support setting SELinux labels. http://b/23530370.
-        selinux_android_restorecon(path, 0);
-
         // fchown clears the setuid bit - restore it if present.
         // Ignore the result of calling fchmod. It's not supported
         // by all filesystems. b/12441485
-- 
2.17.0

